src/clj/dashboard_clj/components/websocket.clj

```diff
@@ -67,3 +81,129 @@
                          :handler           client-ev-handler
                          :webserver-adapter webserver-adapter
                          :options           options}))
+
```

Comment:
Ignore huge comment below.  left it in for future reference if we need to see how some of these messages/events are structured